runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/Work.java

```diff
@@ -101,7 +109,8 @@ private void recordGetWorkStreamLatencies(
     }
   }
 
-  public Collection<Windmill.LatencyAttribution> getLatencyAttributions() {
+  public Collection<Windmill.LatencyAttribution> getLatencyAttributions(
+      Boolean isHeartbeat, String workId, DataflowExecutionStateSampler sampler) {
```

Comment:
unless `isHeartbeat` is nullable, prefer to use the primitive type (`boolean`) to the boxed. go/java-practices/primitives#prefer-unboxed

Reply:
1636e774170b95f004943fa7e80175044f06b206